rtos/ConditionVariable.h

```diff
@@ -194,8 +194,8 @@ class ConditionVariable : private mbed::NonCopyable<ConditionVariable> {
      * variable is notified, or a specific time given by millisec parameter is
      * reached.
      *
-     * @param   millisec  absolute end time referenced to Kernel::get_ms_count()
-     * @return  true if a timeout occurred, false otherwise.
+     * @param   millisec  Absolute end time referenced to `Kernel::get_ms_count()`
+     * @return  True if a timeout occurred, false otherwise.
```

Comment:
This might be the correct fix, but I slightly prefer to view `true` as a keyword here, in which case it doesn't want to be capitalised, and it and `false` should be in backticks instead.

Reply:
That'd be my preferred fix, in fact, but I wasn't sure if they were keywords that actually showed up in the code/returns.

Reply:
I'll fix that.